source/pdps-group-replication.rst

```diff
@@ -0,0 +1,233 @@
+
+The following are the 3 main types of failovers that can occur in database environments:
+
+* **Planned failover**. A planned failover is a failover that has been scheduled in advance or occurs at a regular interval. There can be many reasons for planned failovers including patching, large data operations, retiring existing infrastructure, or simply to test the failover strategy.
+
+* **Unplanned failover**. An unplanned failover is what occurs when a database has unexpectedly become unresponsive or experiences instability. This could also include emergency changes that do not fall under the planned failover cadence or scheduling parameters. Unplanned failovers are generally considered higher risk operations due to the high stress and high potential for either data corruption or data fragmentation.
+
+* **Regional or disaster recovery (DR) failover**. Unplanned failovers still work with the assumption that additional database infrastructure is immediately available and in a usable state. In a regional or DR failover, it is assumed that there is a large scale infrastructure outage which requires the business to move its operations away from its current availability zone.
+
+Maintenance windows
+--------------------------
+
+Major vs Minor maintenance
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Although it may not be obvious at first, not all maintenance activities are created equal and do not have the same dependencies. It is good to separate maintenance that demands downtime or failover from maintenance that can be done without impacting those important stability metrics. When defining these maintenance dependencies, there can be a change in the actual maintenance process that allows for a different cadence.
+
+Maintenance without service interruption
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to cover both major and minor maintenance without service interruption with rolling restart and using proper version upgrade.
+
+Uptime
+--------------------------
+
+When referring to database stability, uptime is likely the largest indicator of stability and oftentimes is the most obvious symptom of an unstable database environment. Uptime is composed of 3 key components and, contrary to common perception, is based on what happens when the database software is not able to take incoming requests rather than maintaining the ability to take requests with errors.
+
+The uptime components are:
+
+* **Recovery Time Objective (RTO)**
+
+  This can be boiled down to a very simple question "How long can the business sustain a database outage?" Once the business is aligned with a goal of a minimum viable recovery time objective, it is much more straightforward to plan and invest in the infrastructure required to meet that requirement. It is important to acknowledge that while everyone desires 100% uptime, there need to be realistic expectations that align with the business needs and not a technical desire.
```

Comment:
This can be 
(What does This refer to?)
to a simple question, "How
Once the business is aligned with a maximum

Reply:
fixed